### PR TITLE
Add e2e coverage for invalid browser url & changing password

### PR DESCRIPTION
**Pull request by cortisiko**
**Description**

_Write a short description of the changes included in this pull request._

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Progresses #???

### CODE REVIEW

File: e2e/specs/delete-wallet.spec.js [hunk 1]
```diff
@@ -12,14 +12,19 @@ import LoginView from '../pages/LoginView';
...
+
 import OnboardingWizardModal from '../pages/modals/OnboardingWizardModal';
 import DeleteWalletModal from '../pages/modals/DeleteWalletModal';
 
 const SECRET_RECOVERY_PHRASE =
 	'ketchup width ladder rent cheap eye torch employ quantum evidence artefact render protect delay wrap identify valley umbrella yard ridge wool swap differ kidney';
 const PASSWORD = `12345678`;
 
-describe('Import wallet with 24 word seedphrase and delete wallet flow', () => {
+describe('Import wallet with 24 word seedphrase, change password then delete wallet flow', () => {
```
Comment (gantunesr) [inline e2e/specs/delete-wallet.spec.js:27]:
```suggestion
describe('Import wallet with 24 word SRP, change password then delete wallet flow', () => {
```

File: app/constants/test-ids.js [hunk 2]
```diff
@@ -20,3 +20,18 @@ export const NETWORK_SCROLL_ID = 'other-networks-scroll';
...
+export const CHANGE_PASSWORD_BUTTON_ID = 'change-password-button-id';
+export const CONFIRM_CHANGE_PASSWORD_INPUT_BOX_ID = 'private-credential-password-text-input';
+
+export const REVEAL_SECRET_RECOVERY_PHRASE_BUTTON_ID = 'reveal-seed-button';
+
+export const SUBMIT_BUTTON_ID = 'submit-button';
+export const BACK_ARROW_BUTTON_ID = 'title-back-arrow-button';
+
+export const IOS_I_UNDERSTAND_BUTTON_ID = 'password-understand-box';
+export const ANDROID_I_UNDERSTAND_BUTTON_ID = 'i-understand-text';
```
Comment (gantunesr) [inline app/constants/test-ids.js:37]: Are we following a naming convention or pattern for this test ids?

File: e2e/pages/Drawer/Browser.js [hunk 3]
```diff
@@ -11,7 +12,8 @@ const HOME_BUTTON_ID = 'home-button';
 const GO_BACK_BUTTON_ID = 'go-back-button';
 const OPTIONS_BUTTON_ID = 'options-button';
 const URL_INPUT_BOX_ID = 'url-input';
-
+const RETURN_HOME_TEXT = strings('webview_error.return_home');
+const BACK_TO_SAFETY_TEXT = strings('phishing.back_to_safety');
 export class Browser {
 	static async tapBrowser() {
```
Comment (gantunesr) [inline e2e/pages/Drawer/Browser.js:18]:
```suggestion
const RETURN_HOME_TEXT = strings('webview_error.return_home');
const BACK_TO_SAFETY_TEXT = strings('phishing.back_to_safety');

export class Browser {
	static async tapBrowser() {
```

File: e2e/pages/Drawer/Browser.js [hunk 4]
```diff
@@ -42,9 +44,11 @@ export class Browser {
 	}
 
 	static async tapBackToSafetyButton() {
-		await TestHelpers.tapByText('Back to safety');
+		await TestHelpers.tapByText(BACK_TO_SAFETY_TEXT);
+	}
+	static async tapReturnHomeButton() {
```
Comment (gantunesr) [inline e2e/pages/Drawer/Browser.js:49]:
```suggestion
	}
	
	static async tapReturnHomeButton() {
```

**Review by gantunesr — APPROVED** [review]
LGTM. Just some minor comments

File: e2e/pages/Drawer/Browser.js [hunk 5]
```diff
@@ -42,9 +45,12 @@ export class Browser {
 	}
 
 	static async tapBackToSafetyButton() {
-		await TestHelpers.tapByText('Back to safety');
+		await TestHelpers.tapByText(BACK_TO_SAFETY_TEXT);
+	}
+
```
Comment (gantunesr) [inline e2e/pages/Drawer/Browser.js:50]:
```suggestion

```

**Review by chrisleewilcox — APPROVED** [review]
LGTM.

**Pull request merged**